packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -78,8 +78,12 @@ class FederationRuntimePlugin {
         runtimePLuginNames.length
           ? Template.asString([
               `${federationGlobal}.initOptions.plugins = ([`,
-              Template.indent(runtimePLuginNames.map((item) => `${item}(),`)),
-              '])',
+              Template.indent(
+                runtimePLuginNames.map(
+                  (item) => `${item} ? ${item}() : false,`,
```

Comment:
we should filter the shaked plugin , and it's content should not be `false`, otherwise the runtime will throw error 
![image](https://github.com/module-federation/universe/assets/41466093/83bf46f5-1901-4d5b-9376-d918c82a16db)

Reply:
like this?
```js
    this.options.plugins = this.options.plugins.reduce((res, plugin) => {
      if(!plugin) return res;
      if (res && !res.find((item) => item.name === plugin.name)) {
        res.push(plugin);
      }
      return res;
    }, pluginRes || []);
    ```

Reply:
Yeah that's nice